src/MfaVerifyController.js

```diff
@@ -100,6 +115,15 @@ function (Okta, Checkbox, BaseLoginController, TOTPForm, YubikeyForm, SecurityQu
           this.options.appState.get('isMfaRequired')) {
         return true;
       }
+      // update auto push cookie after user accepts Okta Verify MFA
+      if (this.options.factorType == 'push') {
+        if (this.settings.get('features.autoPush') && !!this.model.get('autoPush')) {
```

Comment:
it's usually a boolean except when autoPush feature is disabled, this.model.get('autoPush') would return a `undefined`